net-p2p/go-ethereum/Manifest

```diff
@@ -1,2 +1 @@
-DIST go-ethereum-1.7.2.tar.gz 7037907 BLAKE2B eb1e358e42c743a931995e7ee46dc2441356159fc9095e8a886cc6a0495cb2d2e77ee7df4c6dd98bc790397819fc43cdedba6cf28de1e4f3dcdc4dd71106f8e5 SHA512 7bc5957f1523fa5ec67650c4305633aa4c6098a243ef415f9ef56a4e4763eb2ef9f69514ed15810fcdb75f11e87cbd27c4f2af802c32a03be91b37a7b08ea520
```

Comment:
Please split this into two commits: one adding the new version, the other removing old.